instanceidhandler/v1/instanceidhandler.go

```diff
@@ -20,6 +20,12 @@ const (
 	labelFormatKeyContainerName = labelPrefix + "/workload-container-name"
 )
 
+// annotations
+const (
+	annotationPrefix                  = "kubescape.io"
+	AnnotationsFormatForInstanceIDKey = annotationPrefix + "/instance-id"
```

Comment:
maybe we should add the others:
- /image-tag
- /status
- /wlid

Reply:
sure @matthyx , can you send me their keys?

Reply:
WDYT?

Reply:
will do